packages/web/src/config/makeCoreConfig.ts

```diff
@@ -43,5 +43,6 @@ export function makeCoreConfig(browserConfig: BrowserConfig): Config {
     ignoreErrors: browserConfig.ignoreErrors,
     session: browserConfig.session,
     user: browserConfig.user,
+    eventDomain: browserConfig.eventDomain ?? 'browser',
```

Comment:
I think `browser` should be moved to a separate constant and exposed by the package.

Reply:
done